composer.json

```diff
@@ -48,10 +48,11 @@
         "symfony/http-kernel": "^2.8 || ^3.2 || ^4.0",
         "symfony/options-resolver": "^2.8 || ^3.2 || ^4.0",
         "symfony/routing": "^2.8 || ^3.2 || ^4.0",
-        "symfony/security-core": "^2.8 || ^3.2 || ^4.0",
```

Comment:
Do we really not reference any of the classes in that package in our code?

Reply:
https://github.com/sonata-project/SonataMediaBundle/blob/772d0b78fae60b5ad27f2dfc6db47928817b4fe5/src/Security/RolesDownloadStrategy.php#L16

Reply:
Is this an exception? because twig-bundle includes twig-bridge

And framework-bundle includes the 2 http components

IMO we should follow a clear way to declare things on composer.json. To me the rule was, add everything that is referenced on the code explicitly (correct me if im wrong @greg0ire)

To me it makes sense, because sometimes you can say I only want security-bundle 2.8 at minimum. Then you do composer update with prefer-lowest and you can end with security-core < 2.8 because you didnt fix that package too.

wdyt?

Reply:
Fixed